flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/stream/table/TableSinkITCase.scala

```diff
@@ -838,6 +841,44 @@ class TableSinkITCase extends StreamingTestBase {
 
   }
 
+  @Test
+  def testSinkProvider(): Unit = {
```

Comment:
nit: what about `testUnifiedSinkInterface`? Currently, all the SinkRuntimeProviders have keywords of `Sink` and `Provider`.